src/plugins/initiatives/createGraph.js

```diff
@@ -35,6 +37,12 @@ function initiativeNode(initiative: Initiative): Node {
   };
 }
 
+export function initiativeWeight(initiative: Initiative): ?NodeWeight {
+  if (!initiative.weight) return;
+  const {incomplete, complete} = initiative.weight;
+  return !initiative.completed ? incomplete : complete;
```

Comment:
nit: can be simplified as `return initiative.completed ? complete : incomplete`

Reply:
Ah that just happened from trying to keep with the convention of using the chronological order, `incomplete` before `complete`. Which doesn't provide any cognitive load benefit here so I'll update this.

Reply:
Same applies to shortening the variables so going with:
```js
return initiative.completed
  ? initiative.weight.complete
  : initiative.weight.incomplete;
```

Reply:
Done!